lib/puppet/provider/cs_order/crm.rb

```diff
@@ -93,9 +93,13 @@ def flush
     return if @property_hash.empty?
 
     updated = 'order '
-    updated << "#{@property_hash[:name]} #{@property_hash[:score]}: "
+    updated << "#{@property_hash[:name]} "
+    if @property_hash[:score]
+      updated << "#{@property_hash[:score]}: " unless @property_hash[:score].nil?
+    elsif feature? :kindness
+      updated << "#{@property_hash[:kind]}: " unless @property_hash[:kind].nil?
```

Comment:
same here, i reckon

Reply:
Nope, same topic as on resource_discovery: feature and property itself are two different things!
If you look closely they also have different names here (kindness vs kind).
But well, the cs_order type sets the default "Mandatory" and the value is required on the pcs provider, so i will remove the unless here.

Reply:
Closing conversation, since you already approved. (And i did change the code)